src/main/scala/devices/spi/SPIRegs.scala

```diff
@@ -11,6 +11,8 @@ object SPICRs {
   val dsckcs    = 0x2a
   val dintercs  = 0x2c
   val dinterxfr = 0x2e
+  val extradel  = 0x38
```

Comment:
Updated

Reply:
I believe we decided that since we are just adding registers which driver could write and read back to determine if this functionality exists, OK to mark this still as `spi0.`